plugins/module_utils/prism/user_groups.py

```diff
@@ -43,7 +43,7 @@ def _build_spec_project(self, payload, config):
         return payload, None
 
     def _build_spec_user_distinguished_name(self, payload, config):
-        if "ou=" in config:
+        if config[0:3] == "ou=":
```

Comment:
For which case `"ou=" in config` will fail ?

Reply:
in the case of a group being not at the root of the directory...    for example: 
  CN=AdministrativeUsers,OU=admins,DC=contoso,DC=com
  
  the way it is now makes the code think that this group should be loaded as an OU and not a group because it sees OU in the config variable and falsely matches.  It should only be considered an OU if it starts with OU=